sentry-spring-boot-starter/src/main/java/io/sentry/spring/autoconfigure/SentryProperties.java

```diff
@@ -0,0 +1,158 @@
+package io.sentry.spring.autoconfigure;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Configuration for Sentry, example:
+ *
+ * <pre>
+ * sentry:
+ *     enabled: true
+ *     init-default-client: true
+ *     dsn: https://00059966e6224d03a77ea5eca10fbe18@sentry.mycompany.com/14
+ *     release: "1.0.1"
+ *     dist: x86
+ *     environment: staging
+ *     serverName: megaServer
+ *     tags:
+ *         firstTag: Hello
+ *         secondTag: Awesome
+ *     mdcTags: [mdcTagA, mdcTagB]
```

Comment:
Wrong yaml property name format. Must be `mdc-tags`

Reply:
Fixed.